src/SFA.DAS.ApprenticeCommitments.Web/Pages/ConfirmYourIdentity.cshtml.cs

```diff
@@ -68,6 +68,8 @@ public async Task<IActionResult> OnPost([FromServices] AuthenticatedUser user)
                     Email = EmailAddress,
                 });
 
+                await VerifiedUser.ConfirmIdentity(HttpContext);
+
```

Comment:
This page fell over for me locally when I posted back, should we make it a little more tolerant (for local dev), As we do for all the other pages

Reply:
I'm not sure what is going wrong for you, all we do is add the VerifiedUser claim to the signin context.  Maybe we could have a look if you have time?